lib/src/main/java/com/diffplug/spotless/scala/ScalaFmtStep.java

```diff
@@ -64,7 +65,11 @@ public static String defaultVersion() {
 		final FileSignature configSignature;
 
 		State(String version, Provisioner provisioner, @Nullable File configFile) throws IOException {
-			this.jarState = JarState.from(MAVEN_COORDINATE + version, provisioner);
+			String MAVEN_COORDINATE_FOR_VERSION;
+			if (Integer.parseInt(version.substring(0,1)) < 2) MAVEN_COORDINATE_FOR_VERSION = MAVEN_COORDINATE_PRE_2;
+			else MAVEN_COORDINATE_FOR_VERSION = MAVEN_COORDINATE;
```

Comment:
`Integer.parseInt(version.substring(0, 1))` will fail on `20.x`.  I think a regex can be more future proof, [see here](https://github.com/diffplug/spotless/blob/c6a4ae7e4b66e212680c7aff5f64e2304f3bdebe/lib/src/main/java/com/diffplug/spotless/kotlin/KtLintStep.java#L94-L95) for an example.

- Since `MAVEN_COORDINATE_FOR_VERSION` isn't a constant, it should be `mavenCoordinate`
- Use braces in the if-else `if () { } else { }`

Make sure to run `spotlessApply`.  I'm gonna try to figure out why Travis isn't showing built results too...

Reply:
Added a regular expression, renamed mavenCoordinate as indicated and added braces. Ran spotlessApply as part of the commits.